### PR TITLE
Set a different jdk went running forked test (fix #77)

### LINKED ISSUE #77: Find out what we can do to release for jdk6 instead of jdk7

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

**Comment by julienrf** [comment]
That looks good to me! Anyone else want to review? (@lrytz @retronym)

File: build.sbt [hunk 1]
```diff
@@ -32,7 +32,14 @@ lazy val compat = crossProject(JSPlatform, JVMPlatform)
...
   .jvmSettings(
     OsgiKeys.exportPackage := Seq(s"scala.collection.compat.*;version=${version.value}"),
-    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test"
+    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
+    javaHome := {
+      val oldValue = javaHome.value
+      val isOnCi = sys.env.get("CI").isDefined
+
+      if (isOnCi) Some(file(sys.env("JAVA_HOME")))
+      else oldValue
```
Comment (lrytz) [inline build.sbt]: I'm not sure about this change.. it basically relies on a bug, that travis doesn't switch to jdk 6, but JAVA_HOME is set to 6 (https://github.com/travis-ci/travis-ci/issues/9713).

Comment (MasseGuillaume) [reply build.sbt]: @lrytz I force pushed this commit: 

https://github.com/scala/scala-collection-compat/pull/83/commits/3db34829572eadb25c42a8eb947cfcd2ff128625

Do you think this is ok?

**Comment by MasseGuillaume** [comment]
see https://github.com/scala/scala-collection-compat/pull/83

**Comment by julienrf** [comment]
You mean #88?

**Pull request closed without merging**